MDNetworking/MemberImpl/MDReplicatedMember.cs

```diff
@@ -1,69 +1,103 @@
 using Godot;
 using System;
 using System.Reflection;
+using System.Collections.Generic;
 
 namespace MD
 {
     public class MDReplicatedMember
     {
-        public bool ProcessWhilePaused { get; set; } = true;
-
-        public string ReplicationGroup { get; set; } = null;
-
+        public enum Settings
+        {
+            OnValueChangedEvent,
+            Converter
+        }
+        protected const String REPLICATE_METHOD_NAME = "ReplicateClockedValues";
```

Comment:
Do you think this could be a public const on MDReplicator so that it can be set with nameof instead?